README.md

```diff
@@ -4,18 +4,31 @@
   <img src="/images/TasitLogoGrayscale.png" width="200" />
 </div>
 
-### Introduction
+## What is Tasit?
 
-Tasit is a suite of mobile apps for mainstream users that lets them use popular Ethereum dapps.
+Tasit is 2 things: 
 
-Unlike with Status, Coinbase Wallet, Cipher, or other dapp browsers, you can use the dapp via a standalone native mobile app.
+- 1 ) a suite of native mobile apps for mainstream users that lets them use popular Ethereum dapps in the same manner as their other favorite apps (e.g. Instagram, Facebook, Gmail, etc.) 
+- 2 ) a [Javascript SDK](https://github.com/tasitlabs/TasitSDK) that empowers the creation of even more native Ethereum dapps. 
```

Comment:
I think you can empower someone with something, but I don't think you can empower creation itself

Reply:
updated to `empowers developers to create`